mobile/build.gradle

```diff
@@ -20,37 +21,39 @@ repositories {
 
 android {
     compileSdkVersion 23
-    buildToolsVersion "22.0.1"
-
+    buildToolsVersion '25'
     defaultConfig {
         applicationId "com.dexdrip.stephenblack.nightwatch"
-        minSdkVersion 15
+        minSdkVersion 20
```

Comment:
This would exclude KitKat devices that is still the most popular Android version with NightWatch.